src/config/firefox-desktop.js

```diff
@@ -46,28 +55,88 @@ export default {
       },
     },
   },
-  getParamsForQueryString(store) {
+  getParamsForQueryString(storeValue) {
+    /* These parameters will map to a ${key}=${value}&... in the querystring,
+      which is used to convey the view state when the GLAM url is shared with others.
+    */
     return {
-      channel: store.productDimensions.channel,
-      os: store.productDimensions.os,
-      aggregationLevel: store.productDimensions.aggregationLevel,
-      process: store.productDimensions.process,
-      timeHorizon: store.timeHorizon,
-      proportionMetricType: store.proportionMetricType,
-      activeBuckets: store.activeBuckets,
-      visiblePercentiles: store.visiblePercentiles,
+      channel: storeValue.productDimensions.channel,
+      os: storeValue.productDimensions.os,
+      aggregationLevel: storeValue.productDimensions.aggregationLevel,
+      process: storeValue.productDimensions.process,
+      timeHorizon: storeValue.timeHorizon,
+      proportionMetricType: storeValue.proportionMetricType,
+      activeBuckets: storeValue.activeBuckets,
+      visiblePercentiles: storeValue.visiblePercentiles,
```

Comment:
This will need to be updated to include:

```javascript
reference: storeValue.refernece,
```

That might be what the conflict is about.